pkg/config/config_local_test.go

```diff
@@ -588,4 +588,44 @@ var _ = Describe("Config Local", func() {
 		gomega.Expect(err).ToNot(gomega.HaveOccurred())
 		gomega.Expect(config2.Machine.Rosetta).To(gomega.BeFalse())
 	})
+	It("should validate log_path correctly", func() {
+		defConf, err := defaultConfig()
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		gomega.Expect(defConf).NotTo(gomega.BeNil())
+
+		// Test empty path
+		defConf.Containers.LogPath = ""
+		err = defConf.Containers.Validate()
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+		// Test absolute path
+		defConf.Containers.LogPath = "/var/log/containers"
+		err = defConf.Containers.Validate()
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+		// Test path with env var
+		defConf.Containers.LogPath = "$HOME/logs"
```

Comment:
`$HOME` is generally not a safe path, SHELL's expand variables not the application code normally which means if a users put in $HOME it will be used literally.
If you want to expand vars like that it must be done explicitly in our code.